Cargo.toml

```diff
@@ -1,3 +1,6 @@
+[workspace]
+members = ["chat", "providers"]
```

Comment:
Actually, my bad here, must have expressed myself badly, `chat` is a module inside providers :P

So the full path would be `muchat_providers::chat::*`.